solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/health/handlers/SolaceSessionEventHandler.java

```diff
@@ -22,7 +27,10 @@ public void handleEvent(SessionEventArgs eventArgs) {
 		switch (eventArgs.getEvent()) {
 			case RECONNECTED -> this.sessionHealthIndicator.up();
 			case DOWN_ERROR -> this.sessionHealthIndicator.down(eventArgs);
-			case RECONNECTING -> this.sessionHealthIndicator.reconnecting(eventArgs);
+			case RECONNECTING -> {
+				super.handleEvent(eventArgs);
```

Comment:
Instead of calling this here, maybe we should just always call the super method?